components/meter/meter-mixin.js

```diff
@@ -21,6 +21,11 @@ export const MeterMixin = superclass => class extends LocalizeCoreElement(superc
 			 * @type {string}
 			 */
 			text: { type: String },
+			/**
+			 * Hides the text from being displayed beneath the meter
+			 * @type {boolean}
+			 */
+			textHidden: { type: Boolean, attribute: 'text-hidden', reflect: true },
```

Comment:
Looks like `meter-linear` also renders a secondary text element and extends this mixin -- can you apply the same treatment there?